src/coprocessor/endpoint.rs

```diff
@@ -1010,21 +1010,35 @@ mod tests {
 
     #[test]
     fn test_handle_time() {
-        // Asserted that the snapshot can be retrieved in 200ms.
-        const SNAPSHOT_DURATION_MS: i64 = 200;
+        use util::config::ReadableDuration;
 
-        // Asserted that the delay caused by OS scheduling other tasks is smaller than 200ms.
-        // This is mostly for CI.
+        /// Asserted that the snapshot can be retrieved in 500ms.
+        const SNAPSHOT_DURATION_MS: i64 = 500;
+
+        /// Asserted that the delay caused by OS scheduling other tasks is smaller than 200ms.
+        /// This is mostly for CI.
         const HANDLE_ERROR_MS: i64 = 200;
 
+        /// The acceptable error range for a coarse timer. Note that we use CLOCK_MONOTONIC_COARSE
+        /// which can be slewed by time adjustment code (e.g., NTP, PTP).
+        const COARSE_ERROR_MS: i64 = 50;
```

Comment:
Why not set coarse_error_ms to 0 at the very beginning? So that we can get the exact values in CI, and to be honest, I really want to know what they are.😂

Reply:
The time slow may happen randomly 🤔